src/content/extensions/multipleroutes/src/options.js

```diff
@@ -1,97 +1,114 @@
 'use strict';
 
+var pn = chrome.privacy.network;
+var pi = chrome.privacy.IPHandlingPolicy;
+
+var mapPolicyToRadioId = {};
+mapPolicyToRadioId[pi.DEFAULT] = 0;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES] = 1;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_INTERFACE_ONLY] = 2;
+mapPolicyToRadioId[pi.DISABLE_NON_PROXIED_UDP] = 3;
+
+var mapRadioIdToPolicy = {};
+if (pn.webRTCIPHandlingPolicy === undefined) {
+  // radio id => [|webRTCMultipleRoutesEnabled|, |webRTCNonProxiedUdpEnabled|]
+  // The [1] option won't exist if pn.webRTCIPHandlingPolicy is undefined.
+  mapRadioIdToPolicy[0] = [true, true];
+  mapRadioIdToPolicy[2] = [false, true];
+  mapRadioIdToPolicy[3] = [false, false];
+} else {
+  mapRadioIdToPolicy[0] = pi.DEFAULT;
+  mapRadioIdToPolicy[1] = pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES;
+  mapRadioIdToPolicy[2] = pi.DEFAULT_PUBLIC_INTERFACE_ONLY;
+  mapRadioIdToPolicy[3] = pi.DISABLE_NON_PROXIED_UDP;
+}
+
 // Saves options.
 function saveOptions() {
-  var multiRoutes;
-  var nonProxiedUdp;
-  var radios = document.getElementsByName('routeselection');
-  for (var i = 0, length = radios.length; i < length; i++) {
+  var radios = document.getElementsByName('ip_policy_selection');
+  var i = 0;
+  for (; i < radios.length; i++) {
```

Comment:
Putting "var i = " in the for loop is the same thing,  It's JS, so it's all function level anyway.